Language/Glossary/vbe-glossary.md

```diff
@@ -177,7 +177,7 @@ Text added to code that explains how the code works. In Visual Basic, a comment
 
 ## comparison operator
 
-A character or symbol indicating a relationship between two or more values or expressions. These operators include less than (**&lt;**), less than or equal to (**&lt;=**), greater than (**&gt;**), greater than or equal to (**>=**), not equal (**&lt;&gt;**), and equal (**=**). Additional comparison operators include **Is** and **Like**. Note that **Is** and **Like** can't be used as comparison operators in a **Select** **Case** statement.
+Symbol(s) / word indicating a relationship between two or more values or expressions. These operators include less than (**&lt;**), less than or equal to (**&lt;=**), greater than (**&gt;**), greater than or equal to (**>=**), not equal (**&lt;&gt;**), and equal (**=**). Additional comparison operators include **Is** and **Like**. Note that **Is** and **Like** can't be used as comparison operators in a **Select** **Case** statement.
```

Comment:
```suggestion
A language token consisting of one or more characters (letters, symbols) indicating a relationship between two or more values or expressions. These operators include less
```

Reply:
@Linda-Editor @MarkFern, suggested a rewording that captures the need to address the plurality of characters and character types in an operator token, while retaining the technical accuracy of the statement. Thoughts?

Reply:
Note: I made the original comment *before* noticing that the original wording used the layman meaning of "symbol"; this meaning is preserved in the suggested change.

Reply:
Hello **@retailcoder** & **@Linda-Editor**.

Had a very brief review of the programming definition for symbol on Wikipedia [here](https://en.wikipedia.org/wiki/Symbol_(programming)). Not sure whether it is exactly what **@retailcoder** is saying. Is it not something that would be referred to more in the context of the VBA compiler rather than in language-syntax guides?

Anyway, I'm fine with re-wording again. If 'symbol' is what **@retailcoder** is saying, then I can't see the point in mentioning 'character'.

The [_'keyword'_ glossary definition](https://docs.microsoft.com/en-gb/office/vba/language/glossary/vbe-glossary#keyword) (on the same page) is given as:

> A word or symbol recognized as part of the Visual Basic programming language; for example, a statement, function name, or operator.

If 'symbol' encompasses all identifiers, why bother mentioning 'word' in the above definition. Perhaps the writer wanted to say that 'word' & 'symbol' are synonyms for each other but then I don't think the '<' operator is considered a 'word' in programming. I believe the writer is probably referring to what words and symbols mean in a straight-forward, non-programming, non-expert, linguistic context. If this is true, then I think a _keyword_ consisting in two symbols makes sense, as '<=' consists of two symbol characters.

Lexical tokens are described [here](https://en.wikipedia.org/wiki/Lexical_analysis#Token). They are described as being structured as pairs. I'm worried that if we mention tokens, users of the documentation may get confused as it appears more readily associated with parsing phases than programming guides.

---

**@Retailcoder**, in his last comment, agreed that the original change was okay.



I suggest we stick to the original pull request unless anyone else has any other suggestions.
---

---
**@Linda-Editor**, I'll try to create a GitHub _issue_ when the change is small in the future. Is it not easier to make _pull requests_, for you guys? As far as I'm aware, a GitHub _issue_ doesn't formally contain the exact code / documentation changes.

Reply:
@MarkFern In most repos, a PR is okay for submitting changes. But in VBA-Docs, which has over 37K files, each PR merge kicks off a one-hour build (usually builds only take a few minutes). So, if the change is minor, I prefer that it be submitted as an issue, where I can add it to my daily changes (I'm currently doing a page-by-page scrub of this repo).

@retailcoder - can you confirm Mark's decision? Thanks.

Reply:
@MarkFern I agree that introducing the concept of "token" is overly pedantic and potentially confusing for the audience of the documentation. Having spent the past 4-5 years reading various parts of the VBA language specifications, some words (like "symbol" and "keyword") jump *really* at me.

> I believe the writer is probably referring to what words and symbols mean in a straight-forward, non-programming, non-expert, linguistic context.

Upon further reading, this appears to effectively be the case, especially after seeing the glossary definition for "keyword" - just to be clear, operators and standard library functions are absolutely not language keywords. I just wish the official documentation was a little bit more rigorous in its wording, but then again I realize it needs to be accessible to a more general audience.

So how about "A \[Keyword\](glossary-keyword-url) indicating a relationship..."?

Reply:
@MarkFern @retailcoder So just to be clear, we're going with the following text, correct?

Symbol(s) / word indicating a relationship between two or more values or expressions. These operators include less than (**&lt;**), less than or equal to (**&lt;=**), greater than (**&gt;**), greater than or equal to (**>=**), not equal (**&lt;&gt;**), and equal (**=**). Additional comparison operators include **Is** and **Like**. Note that **Is** and **Like** can't be used as comparison operators in a **Select** **Case** statement.

Reply:
@Linda-Editor I'm not sure about losing the leading "A" given all (most?) other definitions start with "A", "An", or "The", but yeah, looks good to me.
Sorry for all the back-and-forth and the dozens of notifications... I did enjoy this discussion very much though!

Reply:
How about: "A symbol or word indicating..."

Reply:
@Linda-Editor That loses @MarkFern's addressing of some operators consisting of multiple "symbols" though, which should remain in the diff - I just can't think of a good way to keep that sentence starting with an article. I didn't mean to derail with that comment!

Reply:
Okay, I'm going with this (I also added a space between the symbols in "> =" and "< ="):

Symbol(s) or a word indicating a relationship between two or more values or expressions. These operators include less than (**&lt;**), less than or equal to (**&lt; =**), greater than (**&gt;**), greater than or equal to (**> =**), not equal (**&lt;&gt;**), and equal (**=**). Additional comparison operators include **Is** and **Like**. Note that **Is** and **Like** can't be used as comparison operators in a **Select Case** statement.

Reply:
Hmm I don't think adding a space is a good idea though... The actual operator really is `<=`, not `< =`...

Reply:
That's what Mark had mentioned in his first comment...that there was a space...but I can close it up...